src/database/migrations/1.0.0/1696782681632-SystemServicesNode.ts

```diff
@@ -0,0 +1,85 @@
+import {MigrationInterface, QueryRunner, Table} from "typeorm";
+
+export class SystemServicesNode1696782681632 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'System')"
+        );
```

Comment:
If you look at the table esquema you will see that `node_type` is the primary key:
```mysql
mysql> show create table fwc_tree_node_types;
+---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table               | Create Table                                                                                                                                                                                                                                                                                                                                          |
+---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| fwc_tree_node_types | CREATE TABLE `fwc_tree_node_types` (
  `node_type` char(3) NOT NULL,
  `obj_type` int(11) DEFAULT NULL,
  `name` varchar(45) DEFAULT NULL,
  `api_call_base` varchar(255) DEFAULT NULL,
  `order_mode` tinyint(1) NOT NULL DEFAULT '1' COMMENT 'Node order: 1-NODE_ORDER, 2 - NAME',
  PRIMARY KEY (`node_type`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 |
+---------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)
```

Then you can not have more than one row with the same `node_type` value.
I suggest you that use `SYS` for the `System` node and `S01`, `S02` and `S03` for the son nodes.

And what about the `order_mode` field? You must set it to `1` like in the nodes for the routing feature.